fsharp-backend/tests/Tests/BwdServer.Tests.fs

```diff
@@ -153,16 +157,17 @@ let parseTest (bytes : byte array) : Test =
            // Allow separation from the next section with a blank line
            request = Array.slice 0 -2 test.request }
 
-// Replace `pattern` in the byte array with `replacement` - both are provided as strings
-// for convenience, but obviously both will be converted to bytes
+/// Replace `pattern` in the byte array with `replacement` - both are provided as strings
+/// for convenience, but obviously both will be converted to bytes
 let replaceByteStrings
   (pattern : string)
   (replacement : string)
   (bytes : byte array)
   : byte array =
   let patternBytes = UTF8.toBytes pattern
   let replacementBytes = UTF8.toBytes replacement |> Array.toList |> List.reverse
-  let list = Array.toList bytes
+  let _list = Array.toList bytes // TODO: can we remove this? it's unused.
```

Comment:
outstanding q - haven't tested with this removed. (maybe it needs to hang around to consume the bytes or something)

Reply:
tests all pass without this - removing.